lib/routes/utils/request_helper.js

```diff
@@ -32,6 +32,11 @@ function shouldEnableSigninConfirmation(account, config, request) {
     return true
   }
 
+  var keysRequested = wantsKeys(request)
+  if (!keysRequested) {
+    return false
+  }
+
```

Comment:
When `shouldEnableSigninConfirmation` returns false, we create the sessionToken in the *verified* state by default:

  https://github.com/mozilla/fxa-auth-server/blob/master/lib/routes/account.js#L436

So IIUC, this change means that any non-keys-requesting login session will be considered verified and can be used to login to sync (assuming you got the keys from somewhere else).

I have to duck out but will try to dig in to the details here in a bit - please don't cut a new tag for this repo until we figure out whether the above is true or not.